### ISSUE TITLE
[Application] Consolidating application statuses

### ISSUE THREAD

**Issue by chrisjluc** [uwblueprint/website#81]
Can we get rid of the `pending` status?

When students/npo apply the default is `pending` but it should probably be `applied`

**Comment by bryanc208** [comment]
The idea behind the `pending` status was to have a bucket of applications that we know to have not been acknowledged by any of our members. Ideally, anything in `pending` should be completely new and when someone comes across a `pending` application, they should triage it to the correct status. 

If we want a new name for the status that is described, we can change that. Perhaps `new` is better? We could maybe even put in a hook to automatically move it to `applied` once at least one member checks out a new application to make it more apparent that the purpose of that status is as described above.

**Comment by chrisjluc** [comment]
Yeah I didn't know pending was for new applicants and that applied meant
somebody had already reviewed it.

This could be communicated more clearly out to the team and maybe the
status labels should be updated.

**Comment by bryanc208** [comment]
Yeah, sounds like a good idea. We could probably change `pending` to `new` and `applied` to `viewed` or `acknowledged` and that should resolve the issue.
